lib/gitrepositoryasync.coffee

```diff
@@ -33,6 +33,23 @@ module.exports = class GitRepositoryAsync
         directoryStatus |= status
       return directoryStatus
 
+  ###
+   * Returns the usual configuration for Git Flow, which includes the prefixes
+   * for each kind of branch.
+   *
+   * @return {Object} Git Flow information
+  ###
+  getFlowConfig: ->
```

Comment:
The `lib/gitrepositoryasync.coffee` should just be a wrapper for the Atom GitRepository with the exact same API.
Move to `lib/gitflow.coffee`